site/pages/dp-auctions/collections/[collectionId].js

```diff
@@ -0,0 +1,235 @@
+
+  return (
+    <Link href={`/dp-auctions/auctions/${auction.id}`} passHref>
+      <tr className="cursor-pointer">
+        <td className="border-2">{auction.id}</td>
+        <td className="border-2">
+          {auction.tokens.map((token) => (
+            <div key={token.address}>
+              <TokenAmount {...token} />
+            </div>
+          ))}
+        </td>
+        <td className="border-2">
+          {auction.status === 'won' || auction.status === 'stopped' ? (
+            '-'
+          ) : (
+            <TokenAmount
+              amount={auction.currentPrice}
+              {...auction.paymentToken}
+            />
+          )}
+        </td>
+        <td className="border-2">
+          {t(auction.status)}
+          {auction.status === 'running' &&
+            ` (${t('ends')} ${DateTime.fromMillis(endTime).toRelative({
+              locale: lang
+            })})`}
+        </td>
+      </tr>
+    </Link>
+  )
+}
+
+// This component renders the list of auctions.
+const DPAuctionsTable = function ({ auctions }) {
+  const { t } = useTranslation('common')
+  const [showEnded, setShowEnded] = useState(false)
+
+  const sortedAuctions = orderBy(
+    auctions.filter((auction) => showEnded || !auction.stopped),
+    ['endBlock', 'id'],
+    ['desc', 'desc']
+  )
+
+  return (
+    <>
+      {sortedAuctions.length ? (
+        <table className="w-full border-collapse">
+          <thead>
+            <tr className="font-bold bg-gray-200">
+              <th className="border-2">{t('id')}</th>
+              <th className="border-2">{t('contains')}</th>
+              <th className="border-2">{t('price')}</th>
+              <th className="border-2">{t('status')}</th>
+            </tr>
+          </thead>
+          <tbody>
+            {sortedAuctions.map((auction) => (
+              <DPAuctionsRow auction={auction} key={auction.id} />
+            ))}
+          </tbody>
+        </table>
+      ) : (
+        <div>{t('no-auctions')}</div>
+      )}
+      {auctions.length ? (
+        <div className="mt-2">
+          <input
+            checked={showEnded}
+            className="mr-1"
+            id="ended-toggle"
+            onChange={() => setShowEnded(!showEnded)}
+            type="checkbox"
+          />
+          <label htmlFor="ended-toggle">{t('show-ended-auctions')}</label>
+        </div>
+      ) : null}
+    </>
+  )
+}
+
+// To allow the Dropdown component to send open/close status and keep track of
+// what collection is selected, the selector component has to be created on the
+// fly keeping the selected collection in the closure.
+const createCollectionSelector = (collectionId) =>
+  function CollectionSelector({ isOpen }) {
+    const { t } = useTranslation('common')
+
+    const rotate = isOpen ? 'transform rotate-180' : ''
+    return (
+      <div className="font-bold">
+        {t('collection')} {collectionId}
```

Comment:
It is better to avoid concatenation of strigs as it may not work in other languages, and use interpolation instead, so each language can place the concatenated word properly
```suggestion
        {t('collection', { collecionId })
```

Reply:
Sounds good. Done in 3ef49b1.